tools/docker-compose.yml

```diff
@@ -149,5 +151,14 @@ services:
       - RUST_LOG=info,topos=debug
       - TARGET_NODES_PATH=/tmp/shared/peer_nodes.json
 
+  autoheal:
```

Comment:
This is not documented. Can you add the rough list of changes as it is expected from the PR template?